test/remote/gateways/remote_flo2cash_test.rb

```diff
@@ -8,7 +8,7 @@ def setup
 
     @amount = 100
     @declined_amount = 110
-    @credit_card = credit_card('5123456789012346', brand: :master, month: 5, year: 2020, verification_value: 111)
+    @credit_card = credit_card('5123456789012346', brand: :master, month: 5, year: Time.now.year + 1, verification_value: 111)
```

Comment:
This is because Flo2Cash was complaining that the expiry date for the CC needs to be in the future.